.github/contributing.md

```diff
@@ -0,0 +1,245 @@
+# Livtours Contribution Code Standards
+
+Livtours follows a series of operating procedures and conventions when contributing code to the codebase. These are outlined below.
+
+## Branch creation and naming conventions utilised
```

Comment:
Here it seems to be taken for granted that a branch will be created to work on an issue. Do we want to setup a convention which instructs how to name the branch when there's no issue involved?

Reply:
Should we always have a related issue?

Reply:
personal opinion, for tangential and technical things creating an issue is just a useless burden (see for example all the PRs I created from a branch `marcosh/...`).

Reply:
Then I guess the branch should be named with the same convention above, but without the `#id-` part